tools/underwater-gps.py

```diff
@@ -60,6 +61,19 @@
                                       # E=Estimated, N=not valid, S=Simulator.
        + "*")                         # Checksum
 
+
+def get_depth():
+    depth = requests.get('http://127.0.0.1:4777/mavlink/VFR_HUD/alt').text
```

Comment:
I believe that is better to organize this function to be inside the class, and move the base path "http..../mavlink" to a class variable.